src/store/index.ts

```diff
@@ -42,7 +42,7 @@ const {
     uploads: [],
   } as State,
   getters: {
-    tables(state: State, getters): string[] {
+    tables(state: State, getters) {
```

Comment:
Why remove the return type?

Reply:
Due to changes brought in from main, the tables being returned are no longer strings, but `Table` objects. I removed the `string[]` so that the return type could be correctly inferred as `Table[]` (as is done with the similar getters below).